python/cutlass/backend/compiler.py

```diff
@@ -465,7 +468,7 @@ def add_module(self, operations, compile_options=None, bypass_cache=False):
                 self._nvcc_compile_options, arch, include_paths, False)
         else:
             cutlass.initialize_sycl_context()
-            arch = "spir64"
+            arch = "intel_gpu_pvc"
```

Comment:
Is this going to break support for BMG?

Reply:
The Python interface was not tested on BMG so far, so it's not really breaking that support as it wasn't supported yet.

I can add logic to detect the architecture and use the correct `arch` in a future PR.

Reply:
If the SPIR-V worked it would have cross-device support, but it doesn't so I guess you're right.